[2024-12-02 06:50:54,640] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: kpi_calculation_pipeline.store_kpi 2024-11-23T02:05:00+00:00 [queued]>
[2024-12-02 06:50:54,691] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: kpi_calculation_pipeline.store_kpi 2024-11-23T02:05:00+00:00 [queued]>
[2024-12-02 06:50:54,692] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-12-02 06:50:54,693] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2024-12-02 06:50:54,694] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-12-02 06:50:54,800] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): store_kpi> on 2024-11-23T02:05:00+00:00
[2024-12-02 06:50:54,805] {standard_task_runner.py:52} INFO - Started process 270 to run task
[2024-12-02 06:50:54,812] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'kpi_calculation_pipeline', 'store_kpi', '2024-11-23T02:05:00+00:00', '--job-id', '405', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/data_injestion_dag/bazefield_kpi_calculation.py', '--cfg-path', '/tmp/tmp7eo3nrpn', '--error-file', '/tmp/tmpjffoijxg']
[2024-12-02 06:50:54,813] {standard_task_runner.py:77} INFO - Job 405: Subtask store_kpi
[2024-12-02 06:50:54,958] {logging_mixin.py:104} INFO - Running <TaskInstance: kpi_calculation_pipeline.store_kpi 2024-11-23T02:05:00+00:00 [running]> on host 59507e5d9338
[2024-12-02 06:50:55,231] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=kpi_calculation_pipeline
AIRFLOW_CTX_TASK_ID=store_kpi
AIRFLOW_CTX_EXECUTION_DATE=2024-11-23T02:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-23T02:05:00+00:00
[2024-12-02 06:50:55,286] {base.py:69} INFO - Using connection to: id: postgres_default. Host: 10.197.48.9, Port: 5432, Schema: kpi_data, Login: rakshit, Password: ***, extra: {}
[2024-12-02 06:50:55,465] {logging_mixin.py:104} INFO - successfully connected
[2024-12-02 06:50:55,545] {python.py:151} INFO - Done. Returned value was: None
[2024-12-02 06:50:55,570] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=kpi_calculation_pipeline, task_id=store_kpi, execution_date=20241123T020500, start_date=20241202T065054, end_date=20241202T065055
[2024-12-02 06:50:55,700] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-02 06:50:55,729] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-12-02 06:50:55,804] {dagrun.py:444} INFO - Marking run <DagRun kpi_calculation_pipeline @ 2024-11-23 02:05:00+00:00: scheduled__2024-11-23T02:05:00+00:00, externally triggered: False> successful
